libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFS20L.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2024
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "AP_RangeFinder_config.h"
+
+#if AP_RANGEFINDER_BENEWAKE_TFS20L_ENABLED
+
+#include "AP_RangeFinder.h"
+#include "AP_RangeFinder_Backend.h"
+
+#define TFS20L_ADDR_DEFAULT              0x10        // TFS20L default device id
```

Comment:
Don't need this in the header

Reply:
Please do not resolve comments unless you've either (a) actually addressed the issue or (b) want to argue the point down (which is an entirely OK approach, push-back is OK)